packages/react-native-bridge/android/build.gradle

```
@@ -1,343 +1,28 @@
 buildscript {
-    def buildGutenbergMobileJSBundle =
-            System.getenv('SUPPRESS_GUTENBERG_MOBILE_JS_BUNDLE_BUILD').asBoolean()
-            ? false : (rootProject.ext.has("buildGutenbergMobileJSBundle")
-                    && rootProject.ext.buildGutenbergMobileJSBundle)
-
-    ext.kotlinVersion = '1.3.61'
-
+    ext {
+        willPublishReactNativeBridgeBinary = gradle.ext.willPublishReactNativeBridgeBinary
```

Nitpick. Given we're already in the context of the `-bridge` package, what do you think of having a shorter name for this to make it easier to read?

```suggestion
        willPublishToBintray = gradle.ext.willPublishReactNativeBridgeBinary
```

This is obviously not something that has to be done in this PR, in particular considering the currently iteration cycle time. Just wanted to bounce the idea 😄 .

I'd say the same suggestion applies to the `willPublishReactNativeAztecBinary` in the `-aztec` package.

---

I don't think this will be a helpful change and I am not sure if it'd work as we expect it to. This is because `willPublishToBintray` can mean anything depending on how the Gradle task is triggered. The values in the `ext` namespace is available across projects, so when the gradle task is triggered from another project (for example when we do `bintrayUpload` by running `:react-native-aztec:bintrayUpload` from `react-native-bridge` project) the same property will be available in the subproject.

Although I agree that these names are verbose, I think it's warranted in this case. We can also re-evaluate these once we move away from Bintray, which will need to happen very soon.

---

> The values in the ext namespace is available across projects, so when the gradle task is triggered from another project (for example when we do bintrayUpload by running :react-native-aztec:bintrayUpload from react-native-bridge project) the same property will be available in the subproject.

I didn't realize that. The longer explicit name is indeed warranted, then.